### DISCUSSION TITLE
add internationalization features

### DISCUSSION THREAD

**Discussion by icanq** [skills/.github#11] [Q&A] [answered]
Hi, greetings. I find this repository very useful in the learning process, currently, I teach my students a lot about version control using Github. is there a possibility to add an internationalization feature? and also can I contribute to the addition/change of content using my country's language. Thank you

**Comment by bpires** [comment, marked as answer]
Hello @icanq, I also plan to provide some localization for Brazilian Portuguese, as the English language is still a barrier for many beginner developers to access quality content. 

It would be good to know the skills team official position on this, but, in my understanding, the Creative Commons Attribution 4.0 International Public License authorizes you to copy and redistribute the material in any medium or format, as well as remix, transform, and build upon the material for any purpose, even commercially.

That means expressly authorizing translations or any other derivative work, as long as you give appropriate credit, provide a link to the license, and reasonably indicate the changes made. [You can find more info about creative commons licenses here](https://creativecommons.org/licenses/by/4.0/)

In this order of ideas, you can start translating right away, and you may use the repo's as templates and follow the [GitHub Skills Quickstart Guide](https://skills.github.com/quickstart) for making your own courses.

**Reply by icanq** [reply]
Thanks for the reply @bpires, will do!
